lib/CustomFieldsFilters/useCustomFieldsSearchableIndexes.js

```diff
@@ -0,0 +1,39 @@
+import { useMemo } from 'react';
+import { useIntl } from 'react-intl';
+
+import { CUSTOM_FIELDS_FILTER, CUSTOM_FIELDS_TYPES } from '../constants/customFields';
```

Comment:
could you please split it into two lines?
__

` from "../constants"` should work as well since you export all from `customFields.js` (`export * from './customFields';`) in `../constants/index.js`, right?